R/faasr_arrow_s3_bucket.R

```diff
@@ -0,0 +1,27 @@
+library("arrow")
+
+
+faasr_arrow_s3_bucket <- function(server_name=.faasr$LoggingServer) {
```

Comment:
Set no default value for the argument "server_name"
`server_name=.faasr$LoggingServer` -> `server_name`